sh_scrapy/compat.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import unicode_literals
```

Comment:
You're right, there're more potential problems with it than benefits, dropped it.